frontend-react/src/routes.js

```diff
@@ -70,6 +71,11 @@ const routes = [
     key: routeKeys.NOTFOUND,
     component: lazy(() => import('./views/NotFound'))
   },
+  {
+    path: '/drive',
+    key: routeKeys.DRIVE,
+    component: lazy(() => import('./views/Add Drive'))
```

Comment:
space? I don't think that's a good idea

```suggestion
    component: lazy(() => import('./views/AddDrive'))
```